logviewer/build.gradle

```diff
@@ -36,35 +36,47 @@ jooq {
 }
 
 dependencies {
-    compileOnly 'org.apache.logging.log4j:log4j-api:2.15.0'
-    runtimeOnly 'org.apache.logging.log4j:log4j-jul:2.15.0'
-    runtimeOnly 'org.apache.logging.log4j:log4j-slf4j-impl:2.15.0'
+    implementation('org.apache.logging.log4j:log4j-api') {
+        version {
+            require '2.16.0'
+        }
+        because 'Log4Shell happend'
+    }
+    runtimeOnly('org.apache.logging.log4j:log4j-core') {
+        version {
+            require '2.16.0'
+        }
+        because 'Log4Shell happend'
+    }
+
+    runtimeOnly 'org.apache.logging.log4j:log4j-jul:2.16.0'
+    runtimeOnly 'org.apache.logging.log4j:log4j-slf4j-impl:2.16.0'
```

Comment:
* Why the different style and why this "because" thing?
* If we already update log4j, might also update it in the base application and not just in logviewer? (or is it already 2.16 there?)
* Small typo, it is "happened" not "happend"

Reply:
Different Style is necessary to force gradle to never use a version lower than 2.16.0, even if transitive dependencies would like a lower version.
"because" Is a reason so that if someone look the dependency tree, it will show this reason why it's set at 2.16+

Can include the 2.16 from the base application in this PR if you want.

Typo removed

Reply:
> Can include the 2.16 from the base application in this PR if you want.

Yes, please. thanks